app/listeners/app.init/initPassport.js

```
@@ -22,7 +22,7 @@ function initPassport (app) {
             .send(userData)
             .end(function (err, resp) {
             if (err) {
-                return done (err,false);
+                return done (null,false);
```

I actually think that was correct, don't we want to return the error if an error occurs?  Maybe, I am misunderstanding.

---

So that is a little tricky...if we return the error the app takes you to a new page that just has the error printed on the screen. Since that isn't exactly what we want I switched it to null so that it would at least take us back to the login page. I might have an idea on how to implement some error handling on that. Give me a little bit to see if I can get it to work.

---

So I have a new commit that adds an error handling component that should send a message to know what the error is. If you read the passport.js documentation at http://passportjs.org/docs/authenticate and scroll down to Verify Call back it talks about pasing an err using done here is for server exceptions where as in failed authentication should be handled using null. Unfortunately, I have yet to figure out how to access that message that is sent back. I will keep working on it but I atm I'm not sure how to do it.